browser/components/MarkdownSplitEditor.styl

```diff
@@ -16,22 +16,14 @@
     z-index 10
     cursor col-resize
 
-body[data-theme="dark"]
-  .root
-    .slider
-      border-left 1px solid $ui-dark-borderColor
+apply-theme(theme)
+  body[data-theme={theme}]
+    .root
+      .slider
+        border-left 1px solid get-theme-var(theme, 'borderColor')
 
-body[data-theme="solarized-dark"]
-  .root
-    .slider
-      border-left 1px solid $ui-solarized-dark-borderColor
+for theme in 'dark' 'dracula' 'solarized-dark'
```

Comment:
Ideally, `'dark' 'dracula' 'solarized-dark'` and `'default' 'white'` should be in the variable `$themes`. But since I don't want to create new variables for the styles, I didn't merge all the styles.